### PR TITLE
Feature/reset

### PR DESCRIPTION
**Pull request by chunfantasy**
This pull request is made for enabling conn_systemreset function, both for v2 and v5.

### CODE REVIEW

File: src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset.h [hunk 1]
```diff
@@ -66,7 +66,7 @@ extern "C" {
  * @retval NRF_SUCCESS          Encoding success.
  * @retval NRF_ERROR_INTERNAL   Encoding failure. Transport error.
  */
-uint32_t conn_systemreset(void);
+// uint32_t conn_systemreset(void);
```
Comment (bencefr) [inline src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset.h]: Commented out code with documentation is not a good idea, remove.

File: src/sd_api_v2/sdk/components/softdevice/s132/headers/ble.h [hunk 2]
```diff
@@ -78,6 +78,7 @@ enum BLE_COMMON_SVCS
   SD_BLE_USER_MEM_REPLY,                /**< User Memory Reply. */
   SD_BLE_OPT_SET,                       /**< Set a BLE option. */
   SD_BLE_OPT_GET,                       /**< Get a BLE option. */
+  CONN_SYSTEMRESET,
```
Comment (bencefr) [inline src/sd_api_v2/sdk/components/softdevice/s132/headers/ble.h]: I see the pattern here, everything starts with SD_. Any reason this shouldn't?

File: src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset.h (hunk 1 above)
Comment (chunfantasy) [reply src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset.h]: This file is now an empty file actually. It was reserved for the reset function and it is now reserved for refactoring the code.

**Comment by chunfantasy** [comment]
Will make a pull request for the refactored version.

**Pull request closed without merging**